packaging/hebart2023/stimulus_set.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+from brainio.stimuli import StimulusSet
+from brainio.packaging import package_stimulus_set
+import pandas as pd
+import numpy as np
+
+stimuli = []
+image_paths = {}
+stimulus_path = '/Users/linussommer/Desktop/Brain-Score-Data/StimulusSet/'
+
+'''
+Dataset Information:
+
+- From Hebart 2023: https://elifesciences.org/articles/82580v1
+
+- Images were taken from the THINGS object concept and image database.
+- The datasset consists of 1,854 diverse object concepts sampled systematically 
+  from concrete picturable and nameable nouns in the American English language.
+
+Fields:
+- image_number: a number {0,1...1853} indicates the image category
+'''
+
+# initial csv to dataframe processing:
+df = pd.read_csv(stimulus_path + 'metadata.csv')
+
+# add image_ids to correspond with the experimental data
+for index, row in df.iterrows():
+    image_id = row.iloc[0] 
+    image_paths[image_id] = stimulus_path + 'reference_images/' + row.iloc[1][1:]
+    stimuli.append({
```

Comment:
this is fine, but just to make sure: is there any more meta information we can include about the stimuli? eg category, positions, etc? We generally try to include as much information as possible.

Reply:
I could include wordnet id and synset see [here](https://osf.io/sz2ux). Do you think this would be useful? Because of different number of sunsets the structure of the DataFrame might become annoying.

Reply:
There is also [this](https://osf.io/xtafs) and [this](https://osf.io/65dku).

Reply:
generally I would say whatever metadata you can add without a huge hassle, add 👍